.github/workflows/broken_link.yml

```diff
@@ -0,0 +1,21 @@
+name: Check Markdown links
+
+on:
+  push:
+    paths:
+    - '**.md'
+  pull_request:
+    paths:
+    - '**.md'
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: 1
+    - uses: gaurav-nelson/github-action-markdown-link-check@0.5.0
+      with:
+        use-quite-mode: 'yes'
```

Comment:
```suggestion
        use-quiet-mode: 'yes'
```
Typo..?

Reply:
🤦‍♂ 

@FriederikeHanssen Please also update to the latest version with this fix. https://github.com/gaurav-nelson/github-action-markdown-link-check/releases/tag/0.6.0

I've also added a custom folder option, so you can specify to only check links in custom folders if that's a requirement.

Reply:
BTW, you could also use `master` if you want to use the latest version always.
```
- uses: gaurav-nelson/github-action-markdown-link-check@master
```

Reply:
@gaurav-nelson : How is the folder-path option different from specifying the paths on push/pull_request: 
```on:
  push:
    paths:
    - 'markdown/**'
  pull_request:
    paths:
    - 'markdown/**'
```